### ISSUE TITLE
Add more list-style type utilities

### ISSUE THREAD

**Issue by DaniTulp** [tailwindcss/discuss#347]
Hi all,

I was wondering if there was any interest in add more list-style-types like `lower-alpha` for an ordered list like:

a. Item 1
b. Item 2
c. Item 3

I was creating an ordered list inside an orderlist and I wanted to be able to differentiate the nesting level easily.

Right now I just added an inline style like: `style="list-style-type: lower-alpha;"`. I know I can also add my own config, but I guess I was just wondering wether we could extend add these by default. Another one I thought could be useful is `lower-roman`.

[Here's a list of all default list-style-types](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type)

If anyone has any other opinions I would like to hear them!

With kind regards,

Dani Tulp

**Comment by hacknug** [comment]
I used to maintain [a plugin that would add those classes](https://github.com/hacknug/tailwindcss-list-style) but it was deprecated because core added support for them: https://tailwindcss.com/docs/list-style-type/#customizing

**Comment by DaniTulp** [comment]
@hacknug 
Yeah I know I can customize them, I was just thinking it could be useful to add a few more to the core. I might be biased though because I use `<ol>` and list-type a ton. I guess I was just wondering what other people thought about it.
